core/cog-shell.c

```diff
@@ -268,7 +272,7 @@ cog_shell_class_init (CogShellClass *klass)
         g_param_spec_string ("home-uri",
                              "Home URI",
                              "URI initially loaded by the Web view",
-                             NULL,
+                             COG_SHELL_DEFAULT_HOME_URI,
```

Comment:
Actually after testing this a bit more, I am not very happy about it. Having a default value for `home-uri` and always loading the URI during `cog_shell_startup()` will add the URI to WebKit's “back-forward list”, which means there will be always one entry (the “no content” page, before loading `about:blank`), which breaks checking for `webkit_web_view_can_go_back()` — it always returns `TRUE` after creation of the Web view, and it should instead return `FALSE` if nothing has been explicitly loaded.

At this point I am leaning towards removing the `home-uri` functionality from `CogShell` and leaving that functionality to the application.